package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "owncloud-docs",
+  "version": "10.0.10",
+  "description": "The ownCloud core documentation, created with Antora and AsciiDoc",
+  "directories": {
+    "doc": "docs"
+  },
+  "dependencies": {
+    "@antora/cli": "^1.1.1",
+    "@antora/site-generator-default": "^1.1.1",
+    "antora-lunr": "v0.1.0",
+    "lunr": "v2.3.3",
```

Comment:
This is a peer dependency and not a direct dependency, right?

Reply:
I had to do [some reading](https://lexi-lambda.github.io/blog/2016/08/24/understanding-the-npm-dependency-model/) (_always good fun_ 👍) to know exactly what you meant. If I understand you correct, yes, it's a peer dependency. And [antora-lunr's package.json](https://github.com/Mogztter/antora-lunr/blob/master/package.json) shows that it's already a dependency. So it seems that I've added it unnecessarily. _Will fix._